crates/bitwarden-exporters/src/export.rs

```diff
@@ -46,7 +46,7 @@ pub(crate) fn export_organization_vault(
     todo!();
 }
 
-/// See [crate::ClientExporters::export_cxf] for more documentation.
+/// See [crate::ExporterClient::export_cxf] for more documentation.
```

Comment:
I'm surprised this didn't cause an error in CI, are we not doing warning-as-errors for cargo doc?